<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frustumForm</class>
 <widget class="QWidget" name="frustumForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>361</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>Входные данные</string>
   </property>
   <widget class="QPushButton" name="applyPushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>321</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Подтвердить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Радиус нижнего основания   --&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Радиус верхнего основания   --&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="radiusDownLineEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="radiusHighLineEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>l-образующая                           --&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lBasedLineEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>170</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Высота                                        --&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="heightLineEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
